src/main/java/com/createment/springone/RouteController.java

```diff
@@ -137,6 +150,18 @@ public String showLastBook(Model model) {
         return "table";
     }
 
+    @GetMapping("/lastAuthor")
+    public String showAuthor(Model model) {
+        String sqlQuery = "SELECT * FROM authors ORDER BY author_id DESC LIMIT 1";
+
+        TableData authorsTableData = populateBooksModel(sqlQuery);
```

Comment:
Ik denk dat je probleem is dat je het booksmodel probeert te populaten met de auteur tabel maar die fields komen niet overeen

Reply:
Check hoe populateBooksModel en populateCustomerModel werken, waarschijnlijk moet je een nieuwe populateAuthorModel maken